backend/src/test-integration/java/com/crm/api/v1/CarControllerIT.java

```diff
@@ -18,41 +18,41 @@
 @SpringBootTest(classes = App.class)
 @AutoConfigureMockMvc
 @WithMockUser
-public class CarControllerIT {
+class CarControllerIT {
 
     @Autowired
     private MockMvc mvc;
 
     @Test
-    public void shouldReturnCorrectResponseStatusWhenCallingCarsEndpoint() throws Exception {
+    void shouldReturnCorrectResponseStatusWhenCallingCarsEndpoint() throws Exception {
         mvc.perform(get("/cars"))
                 .andExpect(status().is(OK.value()));
     }
 
     @Test
-    public void shouldResponseEntityHasPageNumberEqualTo2WhenParamSizeEqualTo1() throws Exception {
+    void shouldResponseEntityHasPageNumberEqualTo2WhenParamSizeEqualTo1() throws Exception {
         mvc.perform(get("/cars?page=1&size=1"))
                 .andExpect(status().is(OK.value()))
                 .andExpect(jsonPath("$.pageable.pageNumber", is(1)))
                 .andExpect(jsonPath("$.pageable.pageSize", is(1)));
     }
 
     @Test
-    public void shouldResponseWithDefaultValues() throws Exception {
+    void shouldResponseWithDefaultValues() throws Exception {
         mvc.perform(get("/cars"))
                 .andExpect(status().is(OK.value()))
                 .andExpect(jsonPath("$.pageable.pageNumber", is(0)))
                 .andExpect(jsonPath("$.pageable.pageSize", is(20)));
     }
 
     @Test
-    public void shouldReturnCorrectResponseStatusWhenCallingCarWithExistingId() throws Exception {
+    void shouldReturnCorrectResponseStatusWhenCallingCarWithExistingId() throws Exception {
         mvc.perform(get("/cars/1"))
                 .andExpect(status().is(OK.value()));
```

Comment:
W nowych testach możemy już odwoływać się do konkretnych wartości ze skryptu data.sql. Myślę że w tym teście można też sprawdzać czy dane wyciągane z bazy zgadzają się z tym co jest w skrypcie

Reply:
To samo w innych testach w których można to sprawdzić

Reply:
Done